lib/resources/r4/financial_transaction_create.yaml

```diff
@@ -0,0 +1,325 @@
+              "coding": [
+                  {
+                      "system": "urn:iso:std:iso:4217",
+                      "code": "USD",
+                      "display": "United States Dollar",
+                      "userSelected": true
+                  }
+              ],
+              "text": "United States Dollar"
+          }
+      }
+    }
+  note: Should be greater than or equal to the value of the financial-transaction-amount. Required on payment transactions when method is Cash.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-tendered-amount
+
+- name: financial-transaction-amount
+  required: 'Yes'
+  cardinality: 0..1
+  type: Money
+  description: The total amount of the FinancialTransaction resource.
+  action: create
+  example: |
+    {
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-amount",
+        "valueMoney": {
+            "value": "123000",
+            "currency": {
+                "coding": [
+                    {
+                        "system": "urn:iso:std:iso:4217",
+                        "code": "USD",
+                        "display": "United States Dollar",
+                        "userSelected": true
+                    }
+                ],
+                "text": "United States Dollar"
+            }
+        }
+    }
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-amount
+
+- name: financial-transaction-allocation
+  required: 'Yes'
+  cardinality: 1..*
+  type: BackboneElement
+  description: How the FinancialTransaction should be applied.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation",
+      "extension": [
+          {
+              "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation-target",
+              "valueReference": {
+                  "reference": "Encounter/31363178"
+              }
+          },
+          {
+              "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation-amount",
+              "valueMoney": {
+                  "value": "400000",
+                  "currency": {
+                      "coding": [
+                          {
+                              "system": "urn:iso:std:iso:4217",
+                              "code": "USD",
+                              "display": "United States Dollar",
+                              "userSelected": true
+                          }
+                      ],
+                      "text": "United States Dollar"
+                  }
+              }
+          }
+      ]
+    }
+  note: Only the first allocation element will be used when creating the resource for this initial release.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation
+
+- name: financial-transaction-allocation-target
```

Comment:
target

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/4824eee1801e3185a37a5b6f312294e683160a0e#diff-26870e1b94fc9955e130f3f940dbe2e7R239

Reply:
@ReaganYuan this still needs to be done

Reply:
isn't it "financial-transaction-target"?

Reply:
No it's just target

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/51acf7d28f90f2f7b2e079e47387bcac290bcb82#diff-26870e1b94fc9955e130f3f940dbe2e7R221

Reply:
The url is just target. Please refer back to a validation step

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/cb886450c7121cafec1a79bf9238bdebd137aff7#diff-26870e1b94fc9955e130f3f940dbe2e7L221

Reply:
+1